### ISSUE TITLE
[BUG] Windows fatal exception: access violation sktime/benchmarking/tests/test_evaluator.py::test_plots

### ISSUE THREAD

**Issue by k1m190r** [alan-turing-institute/sktime#2473]
**Describe the bug**
`sktime/benchmarking/tests/test_evaluator.py::test_plots` fails with  `Windows fatal exception: access violation`

The line where error surfaces is:
`sktime\benchmarking\evaluation.py::plot_boxplots#145`
`fig, ax = plt.subplots(1)`

Solution from https://github.com/matplotlib/matplotlib/issues/22006
downgrade freetype from conda to 2.10.4

`conda install freetype=2.10.4`

**To Reproduce**
````
pytest sktime/benchmarking/tests/test_evaluator.py::test_plots

sktime/benchmarking/tests/test_evaluator.py::test_plots Windows fatal exception: access violation
````

**Expected behavior**
Test passes

**Additional context**
freetype=2.11.0 (anaconda)

**Versions**
Windows, python 3.{7 8 9}

**Comment by fkiraly** [comment]
is this new??

**Comment by fkiraly** [comment]
Is this related to this one:
https://github.com/alan-turing-institute/sktime/issues/2368

**Comment by k1m190r** [comment]
Yes new using current main. I'll check on Mac. I'm not sure if related...

**Comment by fkiraly** [comment]
we should leave it open until resolved

**Comment by k1m190r** [comment]
Oh you mean till it is resolved in matplotlib/freetype?

**Comment by fkiraly** [comment]
yes, imo we should track this until (a) we understand what the error is, and (b) it is explained here, and (c) it is fixed/resolved (upstream or in sktime wherever it needs to be)
